### PR TITLE
Fix bugs from packaging checkin

### PR DESCRIPTION
**Pull request by tbachman**
Commit a02d193190cb0d04ef74733b9139a805ddbcad6d added debian
packaging. That commit had several problems, which this patch
fixes.

Signed-off-by: Thomas Bachman tbachman@yahoo.com

### CODE REVIEW

**Comment by bdemers** [comment]
:+1:

File: tools/packages/debian/build-deb.sh [hunk 1]
```diff
@@ -2,11 +2,11 @@
 # Should be run from the root of the source tree
 # Set env var REVISION to overwrite the 'revision' field in version string
 
-if [ ! -d debian ]; then
+if [ ! -d tools/packages/debian ]; then
```
Comment (amittbose) [inline tools/packages/debian/build-deb.sh]: Nit: how about using a variable to hold the location of the Debian packaging scripts, and use that later:

PKG_SCRIPTS_DIR=tools/packages/debian

Comment (tbachman) [inline tools/packages/debian/build-deb.sh]: Sounds reasonable. Will fix.

Comment (tbachman) [inline tools/packages/debian/build-deb.sh]: Just reposted with this change.

**Pull request merged**